sdk/endpoints/online/managed/debug-online-endpoints-locally-in-visual-studio-code.ipynb

```diff
@@ -0,0 +1,197 @@
+   "source": [
+    "credential = AzureCliCredential()\n",
+    "ml_client = MLClient(\n",
+    "    credential,\n",
+    "    subscription_id=subscription_id,\n",
+    "    resource_group_name=resource_group,\n",
+    "    workspace_name=workspace_name,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "name": "launch-container-4"
+   },
+   "outputs": [],
+   "source": [
+    "deployment = ManagedOnlineDeployment(\n",
+    "    name=\"blue\",\n",
+    "    endpoint_name=endpoint_name,\n",
+    "    model=Model(path=\"../../model-1/model\"),\n",
+    "    code_configuration=CodeConfiguration(\n",
+    "        code=\"../../model-1/onlinescoring\", scoring_script=\"score.py\"\n",
+    "    ),\n",
+    "    environment=Environment(\n",
+    "        conda_file=\"../../model-1/environment/conda.yml\",\n",
+    "        image=\"mcr.microsoft.com/azureml/openmpi3.1.2-ubuntu18.04:20210727.v1\",\n",
+    "    ),\n",
+    "    instance_type=\"Standard_DS2_v2\",\n",
+    "    instance_count=1,\n",
+    ")\n",
+    "\n",
+    "# Note the local and vscode_debug flags\n",
+    "deployment = ml_client.online_deployments.begin_create_or_update(\n",
+    "    deployment, local=True, vscode_debug=True\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "name": "launch-container-5"
+   },
+   "source": [
+    "## 2. Debug your endpoint"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "request_file_path = \"<REQUEST_FILE_PATH>\"\n",
+    "\n",
+    "endpoint.invoke(endpoint_name, request_file_path, local=True)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "name": "launch-container-6"
+   },
+   "outputs": [],
+   "source": [
+    "endpoint = ml_client.online_endpoints.get(endpoint_name, local=True)\n",
+    "endpoint.as_dict()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 3. Edit your endpoint"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "name": "edit-endpoint-1"
+   },
+   "outputs": [],
+   "source": [
+    "new_deployment = ManagedOnlineDeployment(\n",
+    "    name=\"green\",\n",
+    "    endpoint_name=endpoint_name,\n",
+    "    model=Model(path=\"../../model-1/model\"),\n",
+    "    code_configuration=CodeConfiguration(\n",
+    "        code=\"../../model-1/onlinescoring\", scoring_script=\"score.py\"\n",
+    "    ),\n",
+    "    environment=Environment(\n",
+    "        conda_file=\"../../model-1/environment/conda.yml\",\n",
+    "        image=\"mcr.microsoft.com/azureml/openmpi3.1.2-ubuntu18.04:20210727.v1\",\n",
+    "    ),\n",
+    "    instance_type=\"Standard_DS2_v2\",\n",
+    "    instance_count=1,\n",
+    ")\n",
+    "\n",
+    "\n",
+    "ml_client.online_deployments.update(new_deployment, local=True, vscode_debug=True)"
```

Comment:
Does this `ml_client.online_deployments.update` method really exist? Do you mean `ml_client.online_deployments.begin_create_or_update`?
[azure.ai.ml.operations.OnlineDeploymentOperations class | Microsoft Docs](https://docs.microsoft.com/en-us/python/api/azure-ai-ml/azure.ai.ml.operations.onlinedeploymentoperations?view=azure-python-preview)